Procfile

```diff
@@ -1 +0,0 @@
-web: python -m girder --port ${PORT}
```

Comment:
If we do care about Heroku deployment, we should file a bug for the issue.

Regardless, right now, the config is broken, and should be removed. If anyone wants to work on the bug, they can resurrect the old docs from Git's history.